src/theme/custom.scss

```diff
@@ -6,6 +6,7 @@ $white15: rgba(255, 255, 255, 0.15);
 $white20: rgba(255, 255, 255, 0.2);
 $white35: rgba(255, 255, 255, 0.35);
 $white60: rgba(255, 255, 255, 0.6);
+$white70: rgba(255, 255, 255, 0.7);
```

Comment:
With the updated Opacities, We should be able to delete all of the `$whiteXX` and `$blackXX` from our theme. Can you search for anywhere we are using those values and update them accordingly?

Reply:
So, I added this as the combination of the `text-white70` and `text-opacity-70` was the only way I could get the effect to work, same for the `bg-opacity`.